nodes/src/nodes/eval_cobalt/cobalt_evaluator.py

```
@@ -0,0 +1,285 @@
+
+        if not _cobalt_available:
+            debug('cobalt-ai not installed; falling back to basic similarity')
+            return self._fallback_semantic(output, expected, threshold)
+
+        try:
+            evaluator = Evaluator()
+            result = evaluator.evaluate_similarity(output=output, expected=expected)
+            score = float(result.get('score', 0.0)) if isinstance(result, dict) else float(getattr(result, 'score', 0.0))
+            reasoning = result.get('reasoning', '') if isinstance(result, dict) else getattr(result, 'reasoning', '')
+            return self._make_result(score, threshold, reasoning or 'Semantic similarity evaluated', 'semantic')
+        except Exception as e:
```

Ruff (0.15.7)

[warning] Do not catch blind exception: `Exception`

(BLE001)